libs/blocks/global-navigation/utilities/keyboard/index.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable class-methods-use-this */
+import { getNextVisibleItemPosition, getPreviousVisibleItemPosition, selectors } from './utils.js';
+import MainNav from './mainNav.js';
+import { closeAllDropdowns } from '../utilities.js';
+
+const cycleOnOpenSearch = ({ e, isDesktop }) => {
+  const withoutBreadcrumbs = [
+    ...document.querySelectorAll(`
+      ${selectors.brand}, 
+      ${selectors.mainNavToggle},
+      ${selectors.mainNavItems},
+      ${selectors.searchTrigger},
+      ${selectors.searchField},
+      ${selectors.signIn},
+      ${selectors.profileButton},
+      ${selectors.logo}
+  `),
+  ];
+  const first = getNextVisibleItemPosition(-1, withoutBreadcrumbs);
+  const last = getPreviousVisibleItemPosition(withoutBreadcrumbs.length, withoutBreadcrumbs);
+  const openSearch = isDesktop && document.querySelector(selectors.openSearch);
+  if (openSearch && document.activeElement === withoutBreadcrumbs[e.shiftKey ? first : last]) {
+    e.preventDefault();
+    withoutBreadcrumbs[e.shiftKey ? last : first].focus();
+  }
+};
+
+const closeOnClickOutside = (e) => {
+  if (
+    !e.target.closest('.global-navigation')
```

Comment:
we could use `selectors.globalNav`, the value is available here https://github.com/adobecom/milo/blob/keyboard-support/libs/blocks/global-navigation/utilities/keyboard/utils.js#L24